docs/core/config/spam_reporting.md

```diff
@@ -72,7 +72,7 @@ mailbox Spam {
   sieve_script report-spam {
     type = before
     cause = copy
-    path = /etc/dovecot/report-spam.sieve
+    path = /etc/dovecot/sieve/report-spam.sieve
```

Comment:
Please change the commit title to e.g. `config/spam_reporting: Unify sieve script paths`

Reply:
changed